spyder/widgets/sourcecode/editorextensions/closequotes.py

```diff
@@ -3,9 +3,7 @@
 # Copyright © Spyder Project Contributors
 # Licensed under the terms of the MIT License
 # (see spyder/__init__.py for details)
-"""
-This module contains the close quotes editor extension
-"""
+"""This module contains the close quotes editor extension."""
```

Comment:
We could just use `extensions` folder? like `spyder/widgets/sourcecode/extensions/closequotes.py`